### PR TITLE
[ICP-12999] Added vid with correct temperature range

### PR DESCRIPTION
**Pull request by ZWozniakS**
@greens @dkirker @tpmanley 
could you please review?

@KKlimczukS @PKacprowiczS @MGoralczykS @MWierzbinskaS @BJanuszS

### CODE REVIEW

**Review by dkirker — COMMENTED** [review]
@ZWozniakS Was the metadata for `SmartThings-smartthings-TH1300ZB_Sinope_Thermostat` ever committed and uploaded?

**Comment by PKacprowiczS** [comment]
@dkirker version 0.0.8 is on staging, and it's waiting for approval on MR 1079

**Review by greens — APPROVED** [review]

**Comment by dkirker** [comment]
@PKacprowiczS Ha! Yes. Forgot about that one. Looks like it is merged. Could you please request the metadata to production? Both presentation and voice? Also, could you email me a list of all of the vids that need to get deployed and their corresponding tickets? I know there are a couple, at least. I want to make sure that none are forgotten since this process is, unfortunately, still very manual.

**Pull request merged**
